spec/dummy/app/models/user.rb

```diff
@@ -12,10 +10,14 @@ def is_anonymous?
   # OpenStax Accounts "account_user_mapper" methods
 
   def self.account_to_user(acc)
+    raise ArgumentError, 'Account is nil', caller if acc.nil?
+
     acc.is_anonymous? ? AnonymousUser.instance : where(account_id: acc.id).first
   end
 
   def self.user_to_account(user)
+    raise ArgumentError, 'User is nil', caller if user.nil?
+
     user.is_anonymous? ? AnonymousAccount.instance : user.account
```

Comment:
hm, I like the exception but why include `caller`?  That info would be part of the stack trace from the exception, right?

Reply:
Including caller makes it hide the current stack frame since the problem really is in whoever called this.